demos/global-matrix-auth/README.md

```diff
@@ -43,3 +49,13 @@ jenkins:
 
 Some permissions depends on actual plugin-usage.  
 For Example: `Release/*:authenticated` is only available if you _use_ the Release plugin in one of your jobs.
+
+## GitHub Authorization
+
+https://plugins.jenkins.io/github-oauth/
+
+You can configure authorization based on GitHub users, organizations, or teams.
+
+- **username** - specific GitHub username.
+- **organization** - every user that belongs to a specific GitHub organization.
+- **organization*team** - specific GitHub team of a GitHub organization.
```

Comment:
```suggestion
- **organization/team** - specific GitHub team of a GitHub organization.
```

Reply:
@oleg-nenashev I wasn't aware that the '/' format was viable, having used the '*' format in work based on this document: https://plugins.jenkins.io/github-oauth/ (CTRL + F: organization*team).